From 2a2ece0e97425ffa32aba6a5e44b457b815690da Mon Sep 17 00:00:00 2001
From: blobfish <blobfish@gmx.com>
Date: Sun, 28 Mar 2021 16:40:12 -0400
Subject: [PATCH 07/18] ChFi3d_Builder: ChFi3d_ComputeCurves: set tolreached
 for projected, planar pcurves. see following:

    fix for occt bug #0031453
---
 src/ChFi3d/ChFi3d_Builder_0.cxx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ChFi3d/ChFi3d_Builder_0.cxx b/src/ChFi3d/ChFi3d_Builder_0.cxx
index ef17f40eb1..a9438fa0d9 100644
--- a/src/ChFi3d/ChFi3d_Builder_0.cxx
+++ b/src/ChFi3d/ChFi3d_Builder_0.cxx
@@ -3105,6 +3105,7 @@ Standard_Boolean ChFi3d_ComputeCurves(const Handle(Adaptor3d_Surface)&   S1,
       ChFi3d_ProjectPCurv(HC,S1,Pc1,tol3d,tolr1);
       ChFi3d_ProjectPCurv(HC,S2,Pc2,tol3d,tolr2);
       C3d = new Geom_TrimmedCurve(C3d,Udeb,Ufin);
+      tolreached = Min(tolreached,ChFi3d_EvalTolReached(S1,Pc1,S2,Pc2,C3d));
       return Standard_True;
     }
   }
-- 
2.45.0

